peerblock.json

```diff
@@ -49,6 +49,8 @@
                 "url": "https://downloads.sourceforge.net/project/peerblock/PeerBlock_r$matchBuild__Win32_Release.zip"
             }
         },
-        "innosetup": false
+        "pre_install": [
+                "$manifest.innosetup = $false"
```

Comment:
What is the reason for this?

Reply:
There's no stable links for the executable installer, so current version and future updated version are using different package.